tests/utils/result_mapper.py

```
@@ -0,0 +1,120 @@
+            param_name, param_value = list(param.items())[0]
+            nomenclature_params_list.append(
+                f"{param_name}: {param_value}"
+            )
+        nomenclature_params = "\n".join(nomenclature_params_list)
+
+        actual_nomenclature = ""
+        if mapping_result['mappings']:
+            actual_nomenclature = mapping_result['mappings'][0]['nomenclature']
+
+        if actual_nomenclature == "":
+            if mapping_result['similar_mappings']:
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)